CryptoTechReminderSystem/Gateway/HarvestGateway.cs

```diff
@@ -83,5 +85,16 @@ public IList<TimeSheet> RetrieveTimeSheets(DateTimeOffset dateFrom, DateTimeOffs
                 }
             ).ToList(); 
         }
+        
+        private JObject RetrieveATimeSheet(DateTimeOffset dateFrom, DateTimeOffset dateTo, int page)
+        {
+            var address = $"/api/v2/time_entries?from={ToHarvestApiString(dateFrom)}&to={ToHarvestApiString(dateTo)}&page={page}";
+            
```

Comment:
Extra blank line.

Reply:
Resolved in https://github.com/chubberlisk/crypto_tech/pull/33/commits/0b3c026f4cb79597a1f328d37abdaa300c1f4d49.